client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -32,16 +31,9 @@ const EducationEarn = ( { siteSlug } ) => {
 				{
 					calypsoLink: true,
 					url: `/earn/${ siteSlug }`,
-					ModalComponent: VideoModal,
-					modalComponentProps: {
-						isVisible: isModalOpen,
-						onClose: ( event ) => {
-							event.preventDefault();
-							closeModal();
-						},
-						courseSlug: COURSE_SLUGS.PAYMENTS_FEATURES,
-					},
 					onClick: openModal,
+					isOpen: isModalOpen,
+					onClose: closeModal,
```

Comment:
These two new props seems unused:

https://github.com/Automattic/wp-calypso/blob/f9762dbb74be360f1d81716f4e316da0f011c025/client/my-sites/customer-home/cards/education/educational-content/index.jsx#L77

`calypsoLink` and `url` are also unused, but it's unrelated to this PR (I think they were inadvertently added in the previous PR).